blog/2024-06-12-kwoc-annual-report-2023.md

```diff
@@ -0,0 +1,80 @@
+---
+slug: kwoc-annual-report-2023
+title: KWoC 2023 Report
+authors: tejasexpress
+tags:
+  - KOSS
```

Comment:
I don't think there is any need to have a tag named "KOSS" but a KWoC tag would be good, we will probably make more KWoC blogs in the future.

Reply:
Ok the KOSS tag exists so let's keep it but add a KWoC tag.

Reply:
While you're at it, add the tag to the other KWoC blog too.